mcs/class/System/Test/System.Diagnostics/FileVersionInfoTest.cs

```diff
@@ -18,6 +18,7 @@
 using System.Text;
 
 using NUnit.Framework;
+using Assert = NUnit.Framework.Assert;
```

Comment:
Hmm. I'm not honestly sure now. I was getting compile errors when I added it but now it's building fine. Removing.